ONSdigital/florence
# Feature/select a dataset

### What

Update select a dataset page to match UX designs

### How to test

Make sure you have an instance (or instances) that are completed and match to a dataset id in mongodb. Visit http://localhost:8081/florence/datasets and verify functionality works
